ros/processor/process_archive.py

```diff
@@ -0,0 +1,66 @@
+import requests
+import logging
+import pydash as _
+from http import HTTPStatus
+from contextlib import contextmanager
+from tempfile import NamedTemporaryFile
+from insights import extract, rule, run, make_metadata
+from insights.parsers.pmlog_summary import PmLogSummary
+from insights.parsers.lscpu import LsCPU
+from insights.parsers.aws_instance_id import AWSInstanceIdDoc
+from insights.core import dr
+from ros.lib.config import INSIGHTS_EXTRACT_LOGLEVEL
+
+
+LOG = logging.getLogger(__name__)
+dr.log.setLevel(INSIGHTS_EXTRACT_LOGLEVEL)
+
+
+@rule(PmLogSummary, LsCPU, AWSInstanceIdDoc)
+def performance_profile(pmlog_summary, lscpu, aws_instance_id):
+    profile = {}
+    performance_metrics = [
+        'mem.physmem',
+        'mem.util.used',
+        'kernel.all.cpu.user',
+        'kernel.all.cpu.sys',
+        'kernel.all.cpu.nice',
+        'kernel.all.cpu.steal',
+        'kernel.all.cpu.idle',
+        'disk.all.total',
+        'mem.util.cached',
+        'mem.util.bufmem',
+        'mem.util.free'
+        ]
+    profile["total_cpus"] = int(lscpu.info.get('CPUs'))
+    profile["instance_type"] = aws_instance_id.get('instanceType')
+    for i in performance_metrics:
+        profile[i] = _.get(pmlog_summary, f'{i}.val', default='None')
```

Comment:
```suggestion
        profile[i] = _.get(pmlog_summary, f'{i}.val')
```

default value is `None` only, right? if yes, will it be ok to pass only object & path?